src/images_upload_cli/util.py

```diff
@@ -12,49 +12,105 @@
 
 
 class GetEnvError(Exception):
-    """Exception raised for getenv errors."""
+    """Exception raised when an environment variable is not found."""
 
 
 def get_config_path() -> Path:
-    """Get app config path."""
-    return Path(f"{click.get_app_dir('images-upload-cli')}/.env")
+    """
+    Get the path to the app config file.
 
+    Returns:
+        Path: The path to the app config file as a Path object.
+    """
+    app_dir = click.get_app_dir("images-upload-cli")
+    return Path(app_dir) / ".env"
 
-def get_env(key: str) -> str:
-    """Get environment variable or raise an error."""
-    if value := getenv(key):
+
+def get_env(variable: str) -> str:
+    """
+    Get the value of an environment variable.
+
+    Args:
+        variable (str): The name of the environment variable to retrieve.
+
+    Returns:
+        str: The value of the environment variable, if found.
+
+    Raises:
+        GetEnvError: If the environment variable is not found.
+    """
+    if value := getenv(variable):
         return value
 
-    msg = f"Please setup {key} in environment variables or in '{get_config_path()}'."
+    msg = f"Please setup {variable} in environment variables or in '{get_config_path()}'."
     raise GetEnvError(msg)
```

Comment:
**Suggestion:** Consider using a more pythonic way to handle the absence of an environment variable. Instead of raising a custom exception, you can use the `os.environ` dictionary which raises a `KeyError` when the key is not found. This way, you can remove the `GetEnvError` class and the `get_env` function, simplifying your code.
```suggestion
# No need for the GetEnvError class and the get_env function.
# You can directly use os.environ[variable] where needed, which raises a KeyError when the variable is not found.
```